# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/basic/BasicCanvasLayer.h
# Commit: dd00e2da3a0f
# Full Hash: dd00e2da3a0f11a7cc0af9e5ece21a339f4393eb
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-11 04:04:33
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r 69cc1de6f754 -r dd00e2da3a0f gfx/layers/basic/BasicCanvasLayer.h
--- a/gfx/layers/basic/BasicCanvasLayer.h	Wed Jun 10 21:57:19 2020 +0000
+++ b/gfx/layers/basic/BasicCanvasLayer.h	Wed Jun 10 22:21:02 2020 +0000
@@ -35,7 +35,7 @@
     return static_cast<BasicLayerManager*>(mManager);
   }
 
-  CanvasRenderer* CreateCanvasRendererInternal() override;
+  RefPtr<CanvasRenderer> CreateCanvasRendererInternal() override;
 };
 
 }  // namespace layers